### DISCUSSION TITLE
Use of wdio-html-reporter name

### DISCUSSION THREAD

**Discussion by rpii** [webdriverio/webdriverio#7025] [Technical Discussion]
I publish the @rpii/wdio-html-reporter package.  

I would like to take advantage of using the short string name such as "html" in the reporter config section of wdio.conf.

Currently the name of the package would need one of these names:
    @wdio/html-reporter
    wdio-html-reporter

The wdio-html-reporter name is taken by a package that hasnt been updated in 3 years.
Can some suggest or approve a package name that would work?

**Comment by christian-bromann** [comment]
I would recommend to reach out the owner of that package and ask if it would be possible to transfer ownership of it. I've had many times success trying this approach. Otherwise maybe a variation in the name could help.

**Reply by christian-bromann** [reply]
Here is the original repository of that package: https://github.com/mohsen1/wdio-html-reporter

**Comment by rpii** [comment]
Thanks for the response.  I will work on it.

Rich

**Comment by rpii** [comment]
Christiian,

for the short term, I am going with wdio-html-nice-reporter so that
'html-nice' works in webdriverio.  I will negotiate later to get that
package name.
thanks for the support,
the compile error with using the class name was killing us.


Rich

**Reply by christian-bromann** [reply]
What kind of compile error?

**Comment by rpii** [comment]
I put it up for discussion here:

https://github.com/webdriverio/webdriverio/discussions/7004

but no responses
